products/dns/terraform.yaml

```diff
@@ -31,6 +40,22 @@ overrides: !ruby/object:Provider::ResourceOverrides
         exclude: true
       nameServerSet: !ruby/object:Provider::Terraform::PropertyOverride
         exclude: true
+      privateVisibilityConfig.networks: !ruby/object:Provider::Terraform::PropertyOverride
+        is_set: true
+        set_hash_func: |-
+          func(v interface{}) int {
+            raw := v.(map[string]interface{})
+            if url, ok := raw["network_url"]; ok {
+              return selfLinkNameHash(url)
+            }
+            var buf bytes.Buffer
+            schema.SerializeResourceForHash(&buf, raw, resourceDnsManagedZone().Schema["private_visibility_config.networks"].Elem.(*schema.Resource))
```

Comment:
I don't think this line works- Schema is just a regular go map, and so it's not going to have "private_visibility_config.networks" as a key. Now that Riley's change is in, though, you should be able to use `dnsManagedZonePrivateVisibilityConfigNetworksSchema()`

Reply:
I'm inclined to think from my testing in hashicorp/terraform#19658 that we won't be able to support networks as a Set due to the PATCH semantics.